temperature.py

```diff
@@ -0,0 +1,30 @@
+temperature = [16, 17, 18, 19, 20]
+
+
+def valid_check(temperature):
+    for i in temperature:
+        if i > 40:
+            print('The temperature entered is not valid')
+            exit()
+        if i < 16:
+            print('The temperature entered is not valid')
+            exit()
+
+
+def get_avg_temp(temperature):
+    return sum(temperature) / len(temperature)
```

Comment:
1. temperature is a list? If so, maybe temperatures will be a better name, to make it clear it can store several values.
2. Make sure the list is not empty, otherwise you will divide by zero, and it will raise an exception (len([]) will return 0)